File: Data Structures & Algorithms/two-integer-sum/submission-0.py
class Solution:
    def twoSum(self, nums: List[int], target: int) -> List[int]:
        hash_table = defaultdict(list)
        output = [0] * 2
        for index, value in enumerate(nums):
            diff = target - value
            if diff in hash_table:
                output[0] = hash_table[diff][0]
                output[1] = index
                break
            hash_table[value].append(index)
        return output
            
        

File: Data Structures & Algorithms/subsets/submission-0.cs
public class Solution 
{
    public List<List<int>> Subsets(int[] nums) 
    {
        List<List<int>> output = new();
        output.Add(new List<int>());

        for(int i = 0; i < nums.Length; i++)
        {
            int n = nums[i];
            int count = output.Count;
            for(int j = 0; j < count; j++)
            {
                List<int> l = new List<int>(output[j]);
                l.Add(n);
                output.Add(l);
            }
        }

        return output;
    }
}
